scripts/build/Platform/Windows/build_config.json

```diff
@@ -1,4 +1,43 @@
 {
+  "public_metrics_pipe": {
+    "TAGS": [
+      "periodic-incremental-daily-internal"
+    ],
+    "steps": [
+      "test_cpu_profile_internal_metrics",
+      "generate_metrics",
+      "upload_metrics"
+    ]
+  },
+  "generate_metrics": {
+    "TAGS": [],
+    "COMMAND": "python_windows.cmd",
+    "PARAMETERS": {
+      "SCRIPT_PATH": "scripts\\metrics\\ctest_metrics_xml_to_csv.py",
+      "SCRIPT_PARAMETERS": "build\\windows --output-directory build\\windows\\Testing\\csv --branch %BRANCH_NAME% -w"
+    }
+  },
+  "upload_metrics": {
+    "TAGS": [],
+    "COMMAND": "upload_metrics_windows.cmd",
+    "PARAMETERS": {
+      "CSV_DIRECTORY": "build\\windows\\Testing\\csv",
+      "BUCKET": "s3://o3de-metrics/csv/"
```

Comment:
Probably might be better to make this a Jenkins expandable variable like `!O3DE_METRICS_S3_BUCKET!`, as this will not be accessible in other Jenkins environments